apps/src/StudioApp.js

```diff
@@ -394,6 +395,11 @@ StudioApp.prototype.init = function (config) {
         var nonDropletError = false;
         // are we trying to toggle from blocks to text (or the opposite)
         var fromBlocks = this.editor.currentlyUsingBlocks;
+        var event = fromBlocks ? "block-to-text" : "text-to-block";
+        firehoseClient.putRecord(
+          'analysis-events',
+          {study: 'block-and-text-transitions', event: event}
+        );
```

Comment:
I think part of why this feels particularly weird but we're storing two different parts of the same-ish thing in two places. FWIW, I agree that nothing necessarily needs to be changed here. However, I am still curious how one decides whether new logging belongs in Firehose or NR?